spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -325,6 +370,15 @@ follow the pattern `[a-zA-Z_][a-zA-Z0-9_]*`.
 Metric tags are appended to the metric name in curly braces `{` and `}` and are separated by comma.
 Each tag is a key-value-pair in the format of `<key>="<value>"` (the quotes around the value are required).
 
+MicroProfile Metrics timers and histograms expose a Prometheus `summary` type which requires an additional `quantile` tag for certain metrics.
+The `quantile` tag must be included alongside the metrics tags within the curly braces `{` and `}`.
+
+The tag value can be any Unicode character but the following characters must be escaped:
+
+* Backslash (`\`) must be escaped as `\\` (as two characters: `\` and `\`)
+* Double-quotes (`"`) must be escaped as `\"` (as two characters: `\` and `"`)
+* Line feed (`\n`) must be escaped as `\n` (as two characters: `\` and `n`)
+
```

Comment:
should show an example of what the output for at least one metric looks like when there are multiple tags on the metric.